threeplay.go

```diff
@@ -25,17 +25,11 @@ const (
 	HTML OutputFormat = "html"
 )
 
-// HTTPClient interface for http clients used in the 3Play client
-type HTTPClient interface {
-	Get(string) (*http.Response, error)
-	PostForm(string, url.Values) (*http.Response, error)
-}
-
 // Client 3Play Media API client
 type Client struct {
-	apiKey    string
-	apiSecret string
-	client    HTTPClient
+	apiKey     string
+	apiSecret  string
+	HTTPClient *http.Client
```

Comment:
hmm I changed the package so we can have the examples in the _test using `threepplay.NewClient()` etc
I removed the `gock.InterceptClient` calls from the tests and they still pass, maybe its because our client still relies on the default http transport?